API/ev3_sensor.h

```diff
@@ -122,7 +134,7 @@ int SetIRBeaconCH(int sensorPort, int channel);
 /*!
  * @deprecated
 */
-#define initSensors() InitSensors()
```

Comment:
Same. I am all for consistency, but maybe leave in an alias for backwards compatibility? @abke, any thoughts?

Reply:
Yea I think alias will be a good idea here.

Reply:
`InitSensors` and `initSensors` now both call the renamed `SensorsInit`